src/plugin/FileColorPlugin.ts

```diff
@@ -97,25 +97,33 @@ export class FileColorPlugin extends Plugin {
   applyColorStyles = debounce(this.applyColorStylesInternal, 50, true);
 
   private applyColorStylesInternal() {
+    let cssType = this.settings.colorBackground ? 'background' : 'text'
+
     const fileExplorers = this.app.workspace.getLeavesOfType('file-explorer')
     fileExplorers.forEach((fileExplorer) => {
       Object.entries(fileExplorer.view.fileItems).forEach(
         ([path, fileItem]) => {
-          const itemClasses = fileItem.selfEl.classList.value
+          const itemClasses = fileItem.el.classList.value
```

Comment:
This currently gives a type error when compiling:

```
src/plugin/FileColorPlugin.ts:106:40 - error TS2339: Property 'el' does not exist on type '{ selfEl: HTMLDivElement; }'.

106           const itemClasses = fileItem.el.classList.value
                                           ~~

src/plugin/FileColorPlugin.ts:108:22 - error TS7006: Parameter 'cls' implicitly has an 'any' type.

108             .filter((cls) => !cls.startsWith('file-color'))
                         ~~~

src/plugin/FileColorPlugin.ts:123:20 - error TS2339: Property 'el' does not exist on type '{ selfEl: HTMLDivElement; }'.

123           fileItem.el.classList.value = itemClasses.join(' ')
                       ~~
```

We need to update the `global.d.ts` file to type the `el` correctly:

```ts
interface ExplorerView extends View {
  fileItems: Record<
    string,
    {
      selfEl: HTMLDivElement,
      el: HTMLDivElement,
    }
  >
}
```